### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by TheKeymaster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,41 +13,39 @@
 1. [Deployment & Release](#deployment--release)
 1. [Versioning](#versioning)
 
-## Install
+## Installation
```
Comment (TheKeymaster) [inline README.md:16]: You can not have two sections with the same name (the links would not work), therefore I went with `Install`.

File: README.md [hunk 2]
```diff
@@ -73,27 +71,27 @@ Before pushing you may run build to ensure your JS/CSS is built.
...
-Alternatively all tests can be run, by executing
+Alternatively run all tests with
 
 ```
 composer test
 ```
 
 ## Deployment & Release
-1. Update the German and English changelogs in folder `meta/documents`.
+1. Update the german and english change logs in folder `meta/documents`.
```
Comment (TheKeymaster) [inline README.md:87]: Countries, nationalities, and languages may always be capitalized.

File: README.md (hunk 1 above)
Comment (dblaichinger) [reply README.md:16]: How about `Setup`?

File: README.md (hunk 2 above)
Comment (dblaichinger) [reply README.md:87]: Correct please change back.

**Pull request merged**